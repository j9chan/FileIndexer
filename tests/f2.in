tests/book2.in