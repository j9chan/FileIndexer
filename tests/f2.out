4588 the
2199 and
2033 of
1396 to
1285 a
1062 in
780 was
698 that
679 it
552 he
